### PR TITLE
React to AnsiConsole changes

### PR DESCRIPTION
**Pull request by BrennanConroy**
@anurse

### CODE REVIEW

**Comment by ghost** [comment]
Hi **@BrennanConroy**, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Brennan Conroy). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

File: src/Microsoft.Framework.Signing/SignOptions.cs [hunk 1]
```diff
@@ -102,18 +102,20 @@ public X509Certificate2 FindCert(out X509Certificate2Collection includedCerts)
                 {
                     if (query.Count > 1)
                     {
-                        AnsiConsole.Error.WriteLine(@"Multiple certificates were found that meet all the given
+                        var errConsole = AnsiConsole.GetError(false);
+
+                        errConsole.WriteLine(@"Multiple certificates were found that meet all the given
         criteria. Use the -a option to choose the best
         certificate automatically or use the -sha1 option with the hash of the
         desired certificate.");
```
Comment (muratg) [inline src/Microsoft.Framework.Signing/SignOptions.cs:110]: Did you test this output? Looks like it may have unnecessary spaces in it.

**Comment by analogrelay** [comment]
Looks good to me, but @muratg 's comment about spacing does seem relevant. :shipit:

**Comment by BrennanConroy** [comment]
His comment is for stuff that was there before I did anything! I can fix it, but it isn't my fault :D

**Comment by analogrelay** [comment]
Oh, well carry on then :P

**Comment by analogrelay** [comment]
Closing this for now to reduce noise. Signing needs a lot of updating :).

**Pull request closed without merging**